### PR TITLE
Roadmap Styling Updates

### PR DESCRIPTION
**Pull request by khairahscorner**
## Description
@PmS-crypto This PR  would help you familiarise with how to use TailwindCSS to style your pages. Here, I have styled the roadmap to closely match the designs provided. 
Do check them out so you can incorporate similar styles when working on the linear path✨

Also added a basic parameter that can be used to link @PelumiTayo work on the Linkedin cards. However, this does not account for whether the linear/free flowing path was clicked, can be updated later on.

We would have a look during our meeting too.

## Change Type
- [ ] Bug Fix
- [ ] New Feature
- [x] Code Refactor
- [ ] Documentation
- [ ] Other

### CODE REVIEW

File: src/components/FreeflowingCard/index.jsx [hunk 1]
```diff
@@ -1,26 +1,32 @@
...
-                    <PercentageScore />
-                <p className="card-name">{currentCardName}</p>
-                    {/* </div> */}
-                  </CardBody>
-                </Card>
-    )
-}
+  const checkIfLoggedIn = () => {
+    if (localStorage.getItem("jobQuestEmail")) {
+      navigateTo(`/${cardUrl}`); //navigate to taskCard
```
Comment (PelumiTayo) [inline src/components/FreeflowingCard/index.jsx:12]: Okay, this makes sense. I was wondering how the routing to the Task component within App.css would be done now that we have 6 different endpoints that lead to the one component. @khairahscorner 

Also, correct me if I'm wrong, I would get the resource name from the URL and parse information rendered based on that correct? Thank you!

Comment (PelumiTayo) [reply src/components/FreeflowingCard/index.jsx:12]: Nevermind, I now see your comments on my previous pr!

Comment (khairahscorner) [reply src/components/FreeflowingCard/index.jsx:12]: Yes, correct! And well done with the page, you handled the logic really well👍

Comment (PelumiTayo) [reply src/components/FreeflowingCard/index.jsx:12]: Thank you!

**Review by PmS-crypto — APPROVED** [review]

**Comment by PmS-crypto** [comment]
Thanks a lot @khairahscorner 
I now understand the correct way of using classnames for setting up the UI.

**Pull request merged**